docs/building-a-form/common-definitions.md

```diff
@@ -81,63 +86,105 @@ Source: [/src/js/definitions/autosuggest.js](../../src/js/definitions/autosugges
 
 ### Currency
 
-Formats and validates a US currency field. The display includes a leading `$` character. Call this exported function and pass it the label to be used on the field.
+Formats and validates a US currency field that includes a leading `$` character. You can pass this function a label to be used on the field.
+
+![Two currency fields, one with the field selected](../images/definitions/currency.png)
 
 ```js
-import currencyUI from 'us-forms-system/lib/js/definitions/currency';
+import { currencyConfig } from 'us-forms-system/lib/js/definitions/currency';
 
 uiSchema: {
-  payments: currencyUI('Total Payments')
+  currency: currencyConfig.uiSchema('Currency')
+},
+schema: {
+  type: 'object',
+  properties: {
+    currency: currencyConfig.schema()
+  }
 }
 ```
 Source: [/src/js/definitions/currency.js](../../src/js/definitions/currency.js)
 
 ### Current or past dates
 
-The common date field with current or past validation set (i.e., dates in the future are not valid). Call this exported function and pass it the label to be used on the field.
+The common date field with validation warning that dates in the past or future are not valid. You can pass this function a label to be used on the field. Dates must be on or before January 1, 1900.
```

Comment:
Anywhere where it says "You can pass this function a label", I would specify to be "You can pass the `uiSchema` function a label" because there are now 2 functions that are exported by a definition: the `schema()` and the `uiSchema()`. It's important to specify which of the two takes the additional argument.